src/main/java/com/learn/java/inheritance/AbstractSwimmer.java

```diff
@@ -0,0 +1,7 @@
+package com.learn.java.inheritance;
+
+public abstract class AbstractSwimmer implements Swimmer {
+    public void swim(){
+        System.out.println(3);
```

Comment:
modify the run method in order to receive an argument _swimmer_ and print "the _swimmer_ swims"